hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/conf/OzoneManagersCommandHandler.java

```diff
@@ -41,10 +45,16 @@ public class OzoneManagersCommandHandler implements Callable<Void> {
   @Override
   public Void call() throws Exception {
     ConfigurationSource configSource =
-        OzoneConfiguration.of(tool.getConf());
+            OzoneConfiguration.of(tool.getConf());
```

Comment:
I think your IDE is not configured as per Ozone code style. Please use [ozone-style](https://github.com/apache/ozone/blob/764af7cf8dec5cc92e355b2d30339dee6b2c6493/hadoop-ozone/dev-support/intellij/ozone-style.xml#L14) if you are using IntelliJ.